domains/certificates/Query.API/API.IntegrationTests/API.IntegrationTests.csproj

```diff
@@ -8,6 +8,10 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <PropertyGroup Condition="$(CI) == true">
```

Comment:
Where is this set?

Reply:
It is an environment variable and is a default environment variable when running a Github Actions workflow: https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables